README.md

```diff
@@ -1,3 +1,21 @@
+## Archival Notice
+
+📁🗄📜**Repository Archived**
+
+This repository has been archived and will no longer be maintained by Heroku.
+
+**What this means:**
+- Pull requests and issue tracking have been disabled.
+- The project will not receive updates and no new features or security updates will be released.
+- The repository is now read-only.
+- The plugin will still be available to use but may be deprecated.
+
+**Why was it archived?**
+This project is considered complete and active development has concluded.
+
```

Comment:
I will suggest to change this on this particular repo.

Something like:

This project has been deprecated and replaced by [Heroku JVM Application Deployer](https://github.com/heroku/heroku-jvm-application-deployer). Active development has concluded.

Reply:
Done! thank you.